paasta_tools/cli/cmds/status.py

```diff
@@ -70,12 +70,11 @@
 from paasta_tools.utils import PaastaColors
 from paasta_tools.utils import SystemPaastaConfig
 
+
 HTTP_ONLY_INSTANCE_CONFIG: Sequence[Type[InstanceConfig]] = [
     FlinkDeploymentConfig,
     KubernetesDeploymentConfig,
     AdhocJobConfig,
-]
-SSH_ONLY_INSTANCE_CONFIG = [
     ChronosJobConfig,
```

Comment:
Chronos shouldn't be HTTP_ONLY, we want to be able to switch between SSH and HTTP with the use_api_endpoint option.

Reply:
ahh ok, I was under the impression that we wanted to rm ssh all together.